plugins/lang-plan-a/src/main/antlr/PlanAParser.g4

```diff
@@ -34,18 +34,23 @@ statement
     | CONTINUE SEMICOLON?                                                                    # continue
     | BREAK SEMICOLON?                                                                       # break
     | RETURN expression SEMICOLON?                                                           # return
-    | TRY block ( CATCH LP ( TYPE ID ) RP block )+                                           # try
+    | TRY block trap+                                                                        # try
     | THROW expression SEMICOLON?                                                            # throw
     | expression SEMICOLON?                                                                  # expr
     ;
 
 block
-    : LBRACK statement* RBRACK                 # multiple
+    : LBRACK statement+ RBRACK                 # multiple
```

Comment:
Oop! I was reading the change backwards. I thought you were switching from `+` to `*` not the other way around. I understand what you mean about now allowing empty statements some places. Comment withdrawn.